includes/multi-currency/class-user-settings.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * WooCommerce Payments Multi Currency User Settings
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Multi_Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that add Multi Currency settings to user my account page.
+ */
+class User_Settings {
+
+	/**
+	 * Multi-Currency instance.
+	 *
+	 * @var Multi_Currency
+	 */
+	protected $multi_currency;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Multi_Currency $multi_currency The Multi_Currency instance.
+	 */
+	public function __construct( Multi_Currency $multi_currency ) {
+		$this->multi_currency = $multi_currency;
+
+		add_action( 'woocommerce_edit_account_form', [ $this, 'add_default_currency_switch' ] );
+		add_action( 'woocommerce_save_account_details', [ $this, 'save_default_currency' ] );
+	}
+
+	/**
+	 * Add a select to allow user choose default currency in `my account > account details`.
+	 *
+	 * @return void
+	 */
+	public function add_default_currency_switch() {
```

Comment:
I'm wondering whether perhaps we should refer to this as `presentment_currency` or `default_presentment_currency` internally within our code, just to make it clear that it's currency the user will *see* the store prices in, rather than the store's default currency. What do you think?

Reply:
Agree! Good to avoid any future confusion.